lib/go-tc/ats.go

```diff
@@ -0,0 +1,65 @@
+package tc
```

Comment:
Missing license header - this is why the Jenkins job failed.

Reply:
Fixed